### DISCUSSION TITLE
Bad translation : fix it

### DISCUSSION THREAD

**Discussion by gandf** [gandf/slick-rss#29] [General]
Hello,

You can propose new translation directly by answering in this discussion.
For each translation, there is keyword followed by translation.
You must therefore specify here language, keyword and corresponding translation.
![image](https://user-images.githubusercontent.com/4194210/164971192-468e7852-6420-442d-b34d-f6cdc3b93410.png)

If you want to switch back to the English language, activate it in options.
![image](https://user-images.githubusercontent.com/4194210/164994696-4f101868-a66e-426e-945a-a96aa684105c.png)

Thanks

**Comment by desin24** [comment]
messages.json DE
{
  "appName": {
    "message": "Slick RSS by users"
  },
  "appDesc": {
    "message": "Ein voll ausgestatteter RSS-Reader, der vollständig im Browser enthalten ist."
  },
  "appAuthor": {
    "message": "gandf"
  },
  "exportTitle": {
    "message": "Slick RSS : Feeds exportieren"
  },
  "exportMainForm": {
    "message": "Hier ist ein OPML-Export Ihrer Feed-Daten."
  },
  "manageTitle": {
    "message": "Slick RSS : Feeds verwalten"
  },
  "manageManageFeeds": {
    "message": "Feeds verwalten"
  },
  "manageAlertTitle": {
    "message": "Ein Titel ist erforderlich. Benennen Sie es mit etwas Nützlichem wie 'My Awesome Gaming News Feed'."
  },
  "manageAlertUrl": {
    "message": "Eine URL ist erforderlich. Dies ist der vollständige HTTP-Pfad zu Ihrem Feed."
  },
  "manageAlertMaxItemsEmpty": {
    "message": "Max Artikel ist erforderlich. Dies ist die maximale Anzahl von Artikeln, die ich Ihnen für diesen Feed zeigen soll."
  },
  "manageAlertMaxItemsZero": {
    "message": "Ha ha lustiger Mensch. Sie benötigen mindestens 1 max. Artikel, um Ihnen einen Feed anzuzeigen."
  },
  "manageAlertMaxItemsNotItem1": {
    "message": "Sehr lustig, du und ich wissen es beide. '"
  },
  "manageAlertMaxItemsNotItem2": {
    "message": "' ist keine Artikelnummer mit maximalem Wert."
  },
  "manageAlertOrder": {
    "message": "Reihenfolge ist erforderlich. In dieser Reihenfolge zeige ich Ihre Feeds im Viewer an."
  },
  "manageAlertOrderNotItem1": {
    "message": "Sehr lustig, du und ich wissen es beide '"
  },
  "manageAlertOrderNotItem2": {
    "message": "' ist kein Wertauftrag."
  },
  "manageName": {
    "message": "Name"
  },
  "manageUrl": {
    "message": "URL"
  },
  "manageGroup": {
    "message": "Gruppe"
  },
  "manageMaxItems": {
    "message": "Maximale Artikel"
  },
  "manageOrder": {
    "message": "Reihenfolge"
  },
  "importForm": {
    "message": "Fügen Sie den Inhalt Ihrer OPML-Datei in das unten stehende Textfeld ein."
  },
  "importImportFeeds": {
    "message": "Feeds importieren"
  },
  "importTitle": {
    "message": "Slick RSS : Feeds importieren"
  },
  "importAlertNothing": {
    "message": "Nichts zu importieren"
  },
  "importAlertNoOutlineRss": {
    "message": "Es wurden keine Gliederungsknoten vom Typ „rss“ zum Importieren gefunden."
  },
  "importAlertImportedFeeds1": {
    "message": "Importiert "
  },
  "importAlertImportedFeeds2": {
    "message": " Einspeisungen."
  },
  "exportLinkTitle": {
    "message": "Slick RSS OPML Export"
  },
  "optionTitle": {
    "message": "Slick RSS : Optionen"
  },
  "optionFeedBehavior": {
    "message": "Feed-Verhaltensoptionen"
  },
  "optionFeedManagement": {
    "message": "Feed-Management"
  },
  "optionColumns": {
    "message": "Spalten"
  },
  "optionImportFeeds": {
    "message": "Feeds importieren"
  },
  "optionExportFeeds": {
    "message": "Feeds exportieren"
  },
  "optionDefMaxItems": {
    "message": "Standardmaximalelemente"
  },
  "optionFeedDisplay": {
    "message": "Feed-Anzeigeoptionen"
  },
  "optionDarkMode": {
    "message": "Dunkler Modus"
  },
  "optionDescription": {
    "message": "Beschreibung"
  },
  "optionImages": {
    "message": "Bilder"
  },
  "optionObjects": {
    "message": "Objekte (wie Filme)"
  },
  "optionIFrames": {
    "message": "IFrames"
  },
  "optionLoadInBack": {
    "message": "Links im Hintergrund laden"
  },
  "optionShowAllInOne": {
    "message": "Alle Feeds in einem anzeigen"
  },
  "optionShowOnlySummary": {
    "message": "Nur Zusammenfassung anzeigen"
  },
  "optionShowFeedContent": {
    "message": "Feed-Inhalt anzeigen (falls verfügbar)"
  },
  "optionContent": {
    "message": "Inhalt"
  },
  "optionDateFormat": {
    "message": "Datumsformat"
  },
  "optionDone": {
    "message": "OK"
  },
  "optionGMT": {
    "message": "Bewirkt, dass die Ausgabe GMT verwendet. (ansonsten lokal)"
  },
  "optionY4": {
    "message": "4-stelliges Jahr"
  },
  "optionY2": {
    "message": "2-stelliges Jahr"
  },
  "optionMfullText": {
    "message": "Monat (Januar)"
  },
  "optionM3Letters": {
    "message": "Monat (Jan)"
  },
  "optionM0": {
    "message": "Monat (01)"
  },
  "optionM": {
    "message": "Monat"
  },
  "optionDWithSuffix": {
    "message": "Tag (Mo)"
  },
  "optionD0": {
    "message": "Tag (01)"
  },
  "optionD": {
    "message": "Tag"
  },
  "optionH0": {
    "message": "Stunde (01)"
  },
  "optionH": {
    "message": "Std"
  },
  "optionH012": {
    "message": "Stunde im 12-Stunden-Format (01)"
  },
  "optionH12": {
    "message": "Stunde im 12-Stunden-Format"
  },
  "optionMin0": {
    "message": "Minuten (01)"
  },
  "optionMin": {
    "message": "Minuten"
  },
  "optionS0": {
    "message": "Sekunden (01)"
  },
  "optionS": {
    "message": "Sekunden"
  },
  "optionWFullText": {
    "message": "Wochentag (Montag)"
  },
  "optionW3": {
    "message": "Wochentag (Mon)"
  },
  "optionAM": {
    "message": "AM/PM"
  },
  "optionGithubUrl": {
    "message": "GitHub URL"
  },
  "optionFeedUnread": {
    "message": "Optionen für ungelesene Feeds"
  },
  "optionMinutes": {
    "message": " Minuten"
  },
  "optionSecondDisabled": {
    "message": " Sekunden (0 = ausgeschaltet)"
  },
  "optionMarkAllRead": {
    "message": "Alles danach als gelesen markieren"
  },
  "optionMarkAllReadTitleClick": {
    "message": "Nach Titelklick als gelesen markieren"
  },
  "optionReadItemDisplay": {
    "message": "Artikelanzeige lesen"
  },
  "optionGray": {
    "message": "Grau"
  },
  "optionCondensed": {
    "message": "Kondensiert"
  },
  "optionShowTotalUnread": {
    "message": "Gesamt ungelesen anzeigen"
  },
  "optionOnToolbar": {
    "message": "Auf Symbolleiste"
  },
  "optionOnTab": {
    "message": "Auf Registerkarte"
  },
  "optionOnToolbarAndTab": {
    "message": "Auf Symbolleiste & Registerkarte"
  },
  "optionShowTotlaFeedList": {
    "message": "Summen in Feedliste anzeigen"
  },
  "optionPlaySound": {
    "message": "Aktiviere Benachrichtigungen"
  },
  "optionReadLater": {
    "message": "Später lesen Optionen"
  },
  "optionReadLaterSupport": {
    "message": "Lesen Sie später Unterstützung"
  },
  "optionRemoveViewed": {
    "message": "Bei Betrachtung entfernen"
  },
  "optionAlertMaxItemsInvalid": {
    "message": "'Max Items' scheint ungültig zu sein. Dies ist die maximale Anzahl von Elementen in einer Feed-Vorschau und sollte es sein > 0"
  },
  "optionAlertUpdateIntervalInvalid": {
    "message": "'Aktualisierungsintervall' scheint ungültig zu sein. Es ist die Anzahl der Minuten zwischen dem Abrufen der ungelesenen Zählungen."
  },
  "optionAlertUpdateIntervalSup0": {
    "message": "'Aktualisierungsintervall' muss > 0 sein"
  },
  "optionAlertMarkFeedReadInvalid": {
    "message": "'Feed als gelesen markieren nach' scheint ungültig zu sein. Es ist die Anzahl der Sekunden, nachdem Sie einen Feed angesehen haben, bevor er als gelesen markiert wird."
  },
  "optionUpdateInterval": {
    "message": "Updateintervall"
  },
  "optionUseThumbnail": {
    "message": "Vorschaubild in Feeds anzeigen"
  },
  "optionUnlimited": {
    "message": "unbegrenzt"
  },
  "optionFeedsMaxHeight": {
    "message": "Max. Höhe der Feeds"
  },
  "optionFeedsMaxHeightUnits": {
    "message": " px (0 = unbegrenzt)"
  },
  "optionStorageUsage": {
    "message": "Speichernutzung"
  },
  "optionStorageUsageValue1": {
    "message": "Verwenden "
  },
  "optionStorageUsageValue2": {
    "message": " aus "
  },
  "optionStorageUsageValue3": {
    "message": " erhältlich"
  },
  "Bytes": {
    "message": "Bytes"
  },
  "KB": {
    "message": "KB"
  },
  "MB": {
    "message": "MB"
  },
  "GB": {
    "message": "GB"
  },
  "TB": {
    "message": "TB"
  },
  "PB": {
    "message": "PB"
  },
  "EB": {
    "message": "EB"
  },
  "ZB": {
    "message": "ZB"
  },
  "YB": {
    "message": "YB"
  },
  "backReadLater": {
    "message": "Später lesen"
  },
  "backAllFeeds": {
    "message": "Alle Feeds"
  },
  "backItemsMarkedReadLater": {
    "message": "Elemente, die Sie zum späteren Lesen markiert haben"
  },
  "backAllFeedsShow": {
    "message": "Alle Feeds werden hier angezeigt."
  },
  "backNoTitle": {
    "message": "Kein Titel"
  },
  "backErrorXML": {
    "message": "Die Antwort hatte keine gültige responseXML-Eigenschaft."
  },
  "backError200Part1": {
    "message": "Der Status war nicht 200. Er war es "
  },
  "backError200Part2": {
    "message": " und ehrlich gesagt weiß ich nicht wie ich damit umgehen soll. Falls es hilft, war der Statustext '"
  },
  "backError200Part3": {
    "message": "'."
  },
  "backErrorMessage": {
    "message": "Der Feedinhalt hat kein erkanntes Format. Inhalt:"
  },
  "backViewerMarkRead": {
    "message": "Als gelesen markieren"
  },
  "backViewerMarkUnread": {
    "message": "Als ungelesen markieren"
  },
  "openAllFeed": {
    "message": "Alle ungelesenen öffnen"
  },
  "backViewerFeedIssue": {
    "message": "Feed-Probleme"
  },
  "backViewerRefreshing": {
    "message": "Neu laden"
  },
  "backViewerRefreshAll": {
    "message": "Alle neu laden"
  },
  "backViewerMarkAllRead": {
    "message": "Markiere alles Gelesene"
  },
  "backViewerManage": {
    "message": "Verwalten"
  },
  "backViewerRefresh": {
    "message": "Aktualisierung"
  },
  "backViewerOptions": {
    "message": "Optionen"
  },
  "backViewerNoFeed": {
    "message": "Sie haben keine Feeds. Jetzt ist ein guter Zeitpunkt, um etwas hinzuzufügen."
  },
  "backViewerAddFeeds": {
    "message": "Feeds hinzufügen"
  },
  "backViewerFeedSelectedInvalid": {
    "message": "Der ausgewählte Feed scheint ungültig zu sein. Bitte überprüfen Sie die URL."
  },
  "backViewerNerdy": {
    "message": "Nerd-Details"
  },
  "backViewerNoItemInFeed": {
    "message": "Dieser Feed enthält keine Artikel."
  },
  "backViewerFeedMe": {
    "message": "Fütter mich"
  },
  "backViewerLoadingFeed": {
    "message": "Feed wird geladen ..."
  },
  "monthJanuary": {
    "message": "Januar"
  },
  "monthFebruary": {
    "message": "Februar"
  },
  "monthMarch": {
    "message": "März"
  },
  "monthApril": {
    "message": "April"
  },
  "monthMay": {
    "message": "Mai"
  },
  "monthJune": {
    "message": "Juni"
  },
  "monthJuly": {
    "message": "Juli"
  },
  "monthAugust": {
    "message": "August"
  },
  "monthSeptember": {
    "message": "September"
  },
  "monthOctober": {
    "message": "Oktober"
  },
  "monthNovember": {
    "message": "November"
  },
  "monthDecember": {
    "message": "Dezember"
  },
  "daySunday": {
    "message": "Sonntag"
  },
  "dayMonday": {
    "message": "Montag"
  },
  "dayTuesday": {
    "message": "Dienstag"
  },
  "dayWednesday": {
    "message": "Mittwoch"
  },
  "dayThursday": {
    "message": "Donnerstag"
  },
  "dayFriday": {
    "message": "Freitag"
  },
  "daySaturday": {
    "message": "Samstag"
  },
  "daySuffix1": {
    "message": ""
  },
  "daySuffix2": {
    "message": ""
  },
  "daySuffix3": {
    "message": ""
  },
  "daySuffix4": {
    "message": ""
  },
  "daySuffix5": {
    "message": ""
  },
  "optionOff": {
    "message": "Aus"
  },
  "optionOn": {
    "message": "An"
  },
  "close": {
    "message": "Schließen"
  },
  "cancel": {
    "message": "Zurück"
  },
  "save": {
    "message": "Speichern"
  },
  "add": {
    "message": "Hinzufügen"
  },
  "import": {
    "message": "Importieren"
  },
  "export": {
    "message": "Export"
  },
  "optionLevelSearchTag": {
    "message": "Suchtiefen-Tag"
  },
  "optionLevelSearchTags": {
    "message": "Suchtiefen-Tags"
  },
  "optionHelpTranslation": {
    "message": "Hilfe bei der Übersetzung"
  },
  "excludeUnreadCount": {
    "message": "Von der Zählung der ungelesenen Inhalte ausschließen"
  },
  "optionFontSize": {
    "message": "Schriftgröße"
  },
  "optionSize1": {
    "message": "Normal"
  },
  "optionSize2": {
    "message": "Mittel"
  },
  "optionSize3": {
    "message": "Groß"
  },
  "backurlRedirected": {
    "message": "Die URL wurde auf die Adresse umgeleitet "
  },
  "apiTitle": {
    "message": "Slick RSS : API feeds hinzufügen"
  },
  "apiAddFeeds": {
    "message": "Feeds hinzufügen"
  },
  "optionLog": {
    "message": "Aktive Anmeldekonsole"
  },
  "optionVersion": {
    "message": "Ausführung"
  },
  "optionAppName": {
    "message": "Name"
  },
  "optionAppDescription": {
    "message": "Beschreibung"
  },
  "optionContentSummary": {
    "message": "Zusammenfassung des Inhalts"
  },
  "optionShowSummaryFullContent": {
    "message": "Standardmäßig vollständigen Inhalt anzeigen"
  },
  "optionShowSummaryEnable": {
    "message": "Zusammenfassung standardmäßig anzeigen"
  },
  "optionShowSummaryDontShow": {
    "message": "Zusammenfassung nicht anzeigen"
  },
  "backSummaryShow": {
      "message": "Vollständigen Inhalt anzeigen"
  },
  "backSummaryHide": {
      "message": "Zusammenfassung anzeigen"
  },
  "bell": {
      "message": "Artikel bereits gelesen aktualisiert"
  },
  "optionOptionsManagement": {
    "message": "Verwaltung der Optionen"
  },
  "optionImportOptions": {
    "message": "Importoptionen"
  },
  "optionExportOptions": {
    "message": "Exportoptionen"
  },
  "exportOptionsTitle": {
    "message": "Slick RSS : Exportoptionen"
  },
  "exportOptionsForm": {
    "message": "Kopieren Sie den Inhalt Ihrer json Options-Datei in das unten stehende Textfeld."
  },
  "importOptionsTitle": {
    "message": "Slick RSS : Importoptionen"
  },
  "importOptions": {
    "message": "Importoptionen"
  },
  "importOptionsForm": {
    "message": "Fügen Sie den Inhalt Ihrer json Options-Datei in das unten stehende Textfeld ein."
  },
  "exportFeeds": {
    "message": "Feeds exportieren"
  },
  "exportForm": {
    "message": "Kopieren Sie den Inhalt Ihrer OPML-Datei in das unten stehende Textfeld."
  },
  "importAlertError": {
    "message": "Fehler beim Import : "
  },
  "optionshowsavethisfeed": {
    "message": "Schaltfläche zum Speichern eines Feeds anzeigen"
  },
  "showOneFeedTitle": {
    "message": "Slick RSS : Einen Feed anzeigen"
  },
  "lang": {
    "message": "de"
  },
  "optionDontReadOnTitleClick": {
    "message": "Nicht als gelesen markieren, nachdem man auf den Link im Titel eines Artikels geklickt hat"
  },
  "optionForceEnglishLang": {
    "message": "Englische Sprache erzwingen"
  },
  "commentsLink": {
    "message": "Kommentare"
  },
  "optionGetRssFeedUrl": {
    "message": "Erweiterung zum Suchen der Feed-URL auf der Webseite"
  },
  "optionShowToolFindFeed": {
    "message": "Link zum Feed-Suchtool anzeigen"
  },
  "getToolFindFeed": {
    "message": "Feed Finder-Erweiterung"
  },
  "optionUseViewByCategory": {
    "message": "Verwenden Sie die Ansicht nach Kategorien"
  },
  "categoryFilter": {
    "message": "Kategoriefilter"
  },
  "manageAlertCategory": {
    "message": "Kategorie darf nicht leer sein."
  },
  "manageCategories": {
    "message": "Kategorien verwalten"
  },
  "manageCategory": {
    "message": "Kategorie"
  },
  "manageColor": {
    "message": "Farbe verwalten"
  },
  "optionImportCategories": {
    "message": "Kategorien importieren"
  },
  "optionExportCategories": {
    "message": "Kategorien exportieren"
  },
  "optionCategoriesManagement": {
    "message": "Kategorienverwaltung"
  },
  "exportCategoriesTitle": {
    "message": "Slick RSS : Kategorien exportieren"
  },
  "exportCategoriesForm": {
    "message": "Kopieren Sie den Inhalt Ihrer JSON-Kategoriedatei in das Textfeld unten."
  },
  "importCategoriesTitle": {
    "message": "Slick RSS : Kategorien importieren"
  },
  "importCategories": {
    "message": "Kategorien importieren"
  },
  "importCategoriesForm": {
    "message": "Fügen Sie den Inhalt Ihrer JSON-Kategoriedatei in das Textfeld unten ein."
  },
  "NotifyTitle": {
    "message": "Slick RSS : Neuesten Nachrichten"
  },
  "addAuthNotif": {
    "message": "Benachrichtigungsberechtigung hinzufügen"
  },
  "buttonSuggest": {
    "message": "Empfehlen \"Get RSS Feed URL\""
  },
  "DeleteFeed": {
    "message": "Feed löschen"
  },
  "DeleteCategory": {
    "message": "Kategorie löschen"
  },
  "manageCategoriesTitle": {
    "message": "Slick RSS: Kategorien verwalten"
  },
  "manageFontColor": {
    "message": "Schriftfarbe verwalten"
  },
  "manageAlertColor": {
    "message": "Farbe ist nicht gültig."
  },
  "manageAlertFontColor": {
    "message": "Schriftfarbe ist nicht gültig."
  }
}

**Reply by gandf** [reply]
Thanks.
File are updated.
I still don't know when I can put a new version online.
